### ISSUE TITLE
opset-12 support

### ISSUE THREAD

**Issue by guschmue** [onnx/tensorflow-onnx#834]
Status Legend:
*  \+ = Done
* -N = No change needed
* blank = needs investigation

| Operator | Notes | Prev Opset | Status | Collaborator |
| --- | --- | --- | --- | --- |
| ArgMax | New attribute: select_last_index | 11 |  + |  Jparmar |
| ArgMin | New attribute: select_last_index | 11 |  + |  jparmar |
| BatchNormalization | Add optional input: training_mode | 9 | -N (Removed) | jparmar |
| Celu | From pytorch; onnx issue 1121 | N/A |  -N |  jparmar |
| Clip | Support int types | 11 |  +  |  jparmar |
| Constant | Add more attribute for numerical value | 11 |  -N |  jparmar |
| Dropout | Move ratio to input, add attribute seed | 10 | + | jparmar  |
| Einsum | Tf.Einsum | N/A | + | Randy |
| GreaterOrEqual |  | N/A | +  | Randy  |
| Inverse | Inverse of a square matrix | N/A | + | Randy |
| LessOrEqual |  | N/A | + | Randy |
| Max | Support int types | 8 | + |  jparmar |
| MaxPool | Support int8/uint8 | 11 | -N | Randy |
| MeanSquaredDistance | tf.keras.losses.MeanSquaredError | N/A | + | Randy |
| Min | Support int types | 8 | +  |  jparmar |
| NegativeLogLikelihoodLoss | Torch.nn.NLLLoss | N/A | -N | Randy |
| Pow | X , Y different type | 7 | -N |  Randy |
| ReduceMax | Support int8/uint8 | 11 | -N  |  Randy |
| ReduceMin | Support int8/uint8 | 11 |  -N | Randy  |
| SoftmaxCrossEntropyLoss | Torch crossentropyloss | N/A | -N | Randy |
| UnfoldToDepth | Tf.extract_image_patches | N/A | -N (Removed) | Randy |

**Comment by guschmue** [comment]
master has opset-12 enabled in CI.

**Comment by jhagege** [comment]
@RandyShuai can you please explain what is the status of Tf.extract_image_patches ?
I am trying to convert a model using this operator in Tensorflow and getting errors: 

> ERROR - Tensorflow op [inpaint_net/ExtractImagePatches_2: ExtractImagePatches] is not supported

Any planned support for the operator ? 
Alternatively, any suggestion how to work around this ?

Thanks much !

**Comment by RandySheriffH** [comment]
@jhagege : we decided to dump support of the operator since it was removed from ONNX standard, whereas to support your case there are still options like assembling logic in tf2onnx converter with ops at hand.
@TomWildenhain-Microsoft : would u mind help track the request?

**Comment by guschmue** [comment]
This is already tracked here: https://github.com/onnx/tensorflow-onnx/issues/436.
The discussion should move to there and not be on a closed opset-12 issue.
